packages/patternlab/styleguide/source/assets/js/modules/imageFill.js

```diff
@@ -8,7 +8,7 @@ export default (function (window, document, $, undefined) {
 
   function mediaWidth() {
     // Define wrapper width for use.
-    var wrapperWidth = $("#main-content > .main-content--two").width();
+    var wrapperWidth = $("#main-content > .main-content--two").width(); console.log('restored');
```

Comment:
@Javi-er Let's remove this console.log entry if it's not needed

Reply:
thanks @tom-fleming, I just removed that.